package/libs/readline/Makefile

```diff
@@ -70,6 +70,8 @@ endef
 define Package/libreadline/install
 	$(INSTALL_DIR) $(1)/usr/lib
 	$(CP) $(PKG_INSTALL_DIR)/usr/lib/lib{history,readline}.so.* $(1)/usr/lib/
+	$(LN) /usr/lib/libreadline.so.8 $(1)/usr/lib/libreadline.so
```

Comment:
looks wrong.

Reply:
> looks wrong.

Why exactly? I am willing to fix whatever needs to be fixed...

Reply:
It seems to work when installed:

```
root@OpenWRT:~# opkg files libreadline8
Package libreadline8 (8.2-r2) is installed on root and has the following files:
/usr/lib/libreadline.so.8.2
/usr/lib/libhistory.so.8.2
/usr/lib/libreadline.so.8
/usr/lib/libhistory.so
/usr/lib/libhistory.so.8
/usr/lib/libreadline.so
root@OpenWRT:~# ls -l /usr/lib/libreadline.so
lrwxr-xr-x    1 root     root            25 Sep 21 19:31 /usr/lib/libreadline.so -> /usr/lib/libreadline.so.8
```

(And gcc works and finds libreadline too -- which is doesn't without the patch)

Reply:
you have $(INSTALL_DIR) $(1)/usr/lib

so I think you can just do

```
$(LN) ibreadline.so.8 $(1)/usr/lib/libreadline.so
```

Reply:
> you have ( I N S T A L L D I R ) (1)/usr/lib
> 
> so I think you can just do
> 
> ```
> $(LN) libreadline.so.8 $(1)/usr/lib/libreadline.so
> ```

Ah. Ok, I see, make the link relative. I'll do that.

Reply:
Done.